Update exploit-prevention.md (#23727)
DataDog/documentation: content/en/security/application_security/threats/exploit-prevention.md

---

> <div class="alert alert-info">Exploit Prevention is in public beta. Refer to the "library compatibility" section below for supported libraries.</div>

Reviewer: ```suggestion
<div class="alert alert-info">Exploit Prevention is in public beta. See the <a href="#library-compatibility">library compatibility</a> section for supported libraries.</div>
```
